File: java/Arena.java
public class Arena {
    public String presentCircus(String circusName) {
        return "This is the " + circusName + "\n" + "=".repeat(79);
    }

    public String announceAnimal(String animalName, String animalSpecies) {
        return "Next will perform. It is a :" + animalName + "-" + animalSpecies;
    }

    public String displayAnimalPerformance(String performance) {
        return "-> " + performance;
    }
}

File: python/arena.py
class Arena:
    def present_circus(self, circus_name: str):
        return f"This is the {circus_name}\n{'=' * 79}"
    def announce_animal(self, name: str, species: str):
        return f"Next will perform. It is a: {name} - {species}"
    def display_performance(self, performance: str):
        return f"-> {performance}"
